### PR TITLE
New Horizontal Bar Chart

### PR DESCRIPTION
**Pull request by gabycampagna**
For future use.

Will need to turn off all labels at 100x100 on all charts if they will be used as icons in the future. 

To test this please add the follow snippet to visualizer-course-objectives.hbs

{{ilios-chart
  name='horz-bar'
  width=width
  height=height
  data=(await sessionsObjectivesData)
}}

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit 2a20dd0457aab3642be99352494836c5fc11852f

https://deploy-preview-2943--ilios-frontend.netlify.com

File: app/components/chart-horz-bar.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    const width = get(this, 'width');
+    const height = get(this, 'height');
+    const chartWidth = width - margin.left - margin.right;
+    const chartHeight = height - margin.top - margin.bottom;
+    const color = scaleOrdinal(schemeCategory10);
+
+    const x = scaleLinear().range([chartWidth, 0]);
+    const y = scaleBand().range([0, chartHeight]).padding(0.4);
+
+    if (dataOrArray.length === 0) {
```
Comment (jrjohnson) [inline app/components/chart-horz-bar.js]: Once dataOrArray is replace with sample data this should be removed.

Comment (gabycampagna) [inline app/components/chart-horz-bar.js]: Will make those changes and start a new branch for turning off labels for all charts.

**Comment by gabycampagna** [comment]
Test keeps failing. I cannot identify the issue.

**Pull request merged**